crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -0,0 +1,259 @@
+###############################################################################
+# ecumenical.des: This file contains the vaults that place faded altars and
+#                 are a part of the D:1-3 chance group for placing such an
+#                 altar. Vaults that use ecumenical altars for decorative
+#                 reasons should not be in this file.
+#
+# See the section in docs/develop/levels/guidelines.md for guidelines on
+# creating new vaults. Every vault in this file should call
+# ecumenical_altar_setup to set the chance.
+#
+###############################################################################
+
+{{
+function ecumenical_altar_setup(e)
+  e.depth("D:1-3")
+  e.tags("uniq_ecumenical_altar chance_ecumenical_altar extra")
+  e.chance(5000)
+  e.kfeat("_ = altar_ecumenical")
+end
+}}
+
+NAME:   basic_ecumenical_altar
+TAGS:   transparent
+WEIGHT: 200
+: ecumenical_altar_setup(_G)
+MAP
+_
+ENDMAP
+
+NAME:    chequers_ecumenical_altar_simple
+: ecumenical_altar_setup(_G)
```

Comment:
It's customary to have common setup lua (shared by a set of vaults) as final line(s) above `MAP` unless statement order is relevant. Looks like that's not the case here, so would recommend moving this there. If you feel that it's better here, that's fine of course, but it's an established style we have.

Reply:
Moved in the final merge